ocs_ci/utility/rosa.py

```diff
@@ -354,3 +355,24 @@ def delete_oidc_provider(cluster_id):
     """
     cmd = f"rosa delete oidc-provider -c {cluster_id}"
     utils.run_cmd(cmd)
+
+
+def is_odf_addon_installed(cluster_name=None):
+    """
+    Check if the odf addon is installed
+
+    Args:
+        cluster_name (str): The cluster name. The default value is 'config.ENV_DATA["cluster_name"]'
+
+    Returns:
+        bool: True, if the odf addon is installed. False, otherwise
+
+    """
+    cluster_name = cluster_name or config.ENV_DATA["cluster_name"]
+    addon_name = config.ENV_DATA["addon_name"]
```

Comment:
To avoid KeyError please use config.ENV_DATA.get("addon_name") and call get_addon_info only if the addon name is present. Otherwise we will have to do additional check to ensure that this function is called only if the cluster is managed services.